test/test_torch.py

```diff
@@ -15389,6 +15389,88 @@ def test_minmax(self, device, dtype):
         self._test_minmax_helper(lambda x: torch._min_max(x)[0], np.min, device, dtype, skip_indices=True)
         self._test_minmax_helper(lambda x: torch._min_max(x)[1], np.max, device, dtype, skip_indices=True)
 
+    @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
+    @dtypes(*(torch.testing.get_all_int_dtypes() + [torch.bool]))
+    def test_maximum_minimum_int_and_bool(self, device, dtype):
```

Comment:
This test is already very good. You could improve it by generating the NumPy arrays using https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.integers.html?highlight=integers#numpy.random.Generator.integers instead of handwriting a tuple of values. The arrays can then be converted to torch tensors with `torch.from_numpy(a).to(device=device)`.

Reply:
Thanks for the kind suggestions, I have updated `test_maximum_minimum_int_and_bool` to let NumPy generate random inputs for me.